.github/workflows/deploy-beta.yml

```diff
@@ -31,39 +31,53 @@ jobs:
           cache-name: cache-node_modules
         with:
           path: node_modules
-          key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock')}}
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('yarn.lock')}}
           restore-keys: |
-            ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock')}}
+            ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('yarn.lock')}}
             ${{ runner.os }}-${{ env.cache-name }}-
 
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
+        run: echo "::set-output name=dir::$(yarn cache dir)"
 
       - name: Cache yarn
         uses: actions/cache@v2
         env:
           cache-name: cache-yarn
         with:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock') }}
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
           restore-keys: |
-            ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('**/yarn.lock') }}
+            ${{ runner.os }}-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
             ${{ runner.os }}-${{ env.cache-name }}-
 
       - name: Install
         run: yarn install --frozen-lockfile --prefer-offline
 
-      - name: Build and deploy
-        run: ./build/ga_build.sh
+      - name: Build
+        run: yarn run build-beta
         env:
-          PT_PROJECT_TOKEN: ${{ secrets.PT_PROJECT_TOKEN }}
           WEB_API_KEY: ${{ secrets.WEB_API_KEY }}
           WEB_OAUTH_CLIENT_ID: ${{ secrets.WEB_OAUTH_CLIENT_ID }}
           WEB_OAUTH_CLIENT_SECRET: ${{ secrets.WEB_OAUTH_CLIENT_SECRET }}
           DIM_API_KEY: ${{ secrets.DIM_API_KEY }}
-          CLOUDFLARE_KEY: ${{ secrets.CLOUDFLARE_KEY }}
-          CLOUDFLARE_EMAIL: ${{ secrets.CLOUDFLARE_EMAIL }}
+
+      - name: Deploy
+        run: npx grunt publish_beta
+        env:
           REMOTE_HOST: ${{ secrets.REMOTE_HOST }}
           REMOTE_PATH: beta.destinyitemmanager.com
+
+      - name: Purge CloudFlare cache
+        run: ./build/purge-cloudflare-beta.sh
```

Comment:
https://github.com/marketplace/actions/cloudflare-purge-cache

Reply:
Yeah, I looked at both of these actions, and they seemed harder to use than what we already had.